README.md

```diff
@@ -184,9 +171,9 @@ Click the "Load Unpacked" button (top left corner of browser window)
 Navigate to the directory under browser-extensions which contains a manifest.json file and click "Select"
 ```
 
-Now you can add `@firebolt-js/sdk` in your app's `package.json` file, import `@firebolt-js/sdk` in your code, and make calls using the Firebolt core SDK (or use the manage and/or discovery SDKs too). If you pass the appropriate query string parameter(s) (see [docs/UsageWithinApps.md](./docs/UsageWithinApps.md)), these SDK calls will get sent to the Mock Firebolt server and it will reply as you configure it to do so.
+Now you can access core sdk from HTTP by adding `"url": "https://meta.rdkcentral.com/firebolt/api/"` in mf.config.SAMPLE.json and copying to mf.config.json,If you pass the appropriate query string parameter(s) (see [docs/UsageWithinApps.md](./docs/UsageWithinApps.md)), these SDK calls will get sent to the Mock Firebolt server and it will reply as you configure it to do so.
 
-See the section "Important Dependency Notes" above for details about which versions of `@firebolt-js/sdk` (and other SDKs) support Mock Firebolt.
+See the section "Important Dependency Notes" above for details about which version of SDK support Mock Firebolt.
```

Comment:
If we delete that section (see earlier comment), we should delete this note.

Reply:
Addressed